### DISCUSSION TITLE
(不具合)限定共有記事の購読を解除できない

### DISCUSSION THREAD

**Discussion by nnabeyang** [increments/qiita-discussions#208] [要望・アイデア・不具合]
限定共有投稿記事にコメントする機会があったため、購読状態になりました。その後、「記事の購読を解除する」を選択したのですが、『404 Not Found』が表示されます。「記事の購読を解除する」が「記事を購読する」に変化していないため、購読解除できていないようです。

**Comment by tanaka-asami415** [comment]
@nnabeyang 

ご意見ありがとうございます🙏
こちら不具合のようです。原因がすぐにわからないため、調査してご報告します。

こちらにissueを立てております。
https://github.com/increments/qiita-discussions/issues/209

**Comment by wataru86** [comment]
@nnabeyang 
大変お待たせしました🙇
限定共有記事の購読解除を行いました。
また、不具合により限定共有記事を購読できる状態になっていたため、購読できないように修正しました。
